soc/bflb/bl61x/soc.c

```diff
@@ -19,10 +19,70 @@
 #include <hbn_reg.h>
 #include <pds_reg.h>
 
+#define SYSMAP_BASE 0xEFFFF000UL
```

Comment:
Hi,

I'm fine with the changes but you need to add the details in the commit body. We talked a lot of time until find the solution and this should be documented in this case.

I think this explains very well https://cbea.ms/git-commit/

BTW, you defined the system_sysmap_init() but it is not using it.

Reply:
And i confirm that hasnt fixed it. Will look into generating the limited cache invalidate insn.